mino/minogrpc/tracing/tracers.go

```diff
@@ -0,0 +1,77 @@
+package tracing
+
+import (
+	"io"
+	"sync"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	_ "github.com/uber/jaeger-client-go"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+)
+
+var (
+	ProtocolTag = "protocol"
+)
+
+type key int
```

Comment:
is it necessary to define and use that type ?

Reply:
IIRC it had to do with the linter complaining about possible key clashes in context.Value